src/monorepo-workflow-operations.test.ts

```diff
@@ -1,1788 +1,1214 @@
+              sandbox,
+              doesReleaseSpecFileExist: false,
+              isEditorAvailable: false,
             });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: true,
+            today,
+            stdout,
+            stderr,
+          });
+
+          expect(executeReleasePlanSpy).not.toHaveBeenCalled();
+        });
+      });
+
+      it('does not try to create a new branch', async () => {
+        await withSandbox(async (sandbox) => {
+          const {
+            project,
+            today,
+            stdout,
+            stderr,
+            captureChangesInReleaseBranchSpy,
+          } = await setupFollowMonorepoWorkflow({
+            sandbox,
+            doesReleaseSpecFileExist: false,
+            isEditorAvailable: false,
+          });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: true,
+            today,
+            stdout,
+            stderr,
+          });
+
+          expect(captureChangesInReleaseBranchSpy).not.toHaveBeenCalled();
+        });
+      });
+
+      it('prints a message', async () => {
+        await withSandbox(async (sandbox) => {
+          const { project, today, stdout, stderr } =
+            await setupFollowMonorepoWorkflow({
+              sandbox,
+              doesReleaseSpecFileExist: false,
+              isEditorAvailable: false,
+            });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: true,
+            today,
+            stdout,
+            stderr,
+          });
+
+          expect(stdout.data()[0]).toMatch(
+            /^A template has been generated that specifies this release/u,
+          );
+        });
+      });
+
+      it('does not remove the release spec file', async () => {
+        await withSandbox(async (sandbox) => {
+          const { project, today, stdout, stderr, releaseSpecificationPath } =
+            await setupFollowMonorepoWorkflow({
+              sandbox,
+              doesReleaseSpecFileExist: false,
+              isEditorAvailable: false,
+              errorUponEditingReleaseSpec: null,
+            });
+
+          await followMonorepoWorkflow({
+            project,
+            tempDirectoryPath: sandbox.directoryPath,
+            firstRemovingExistingReleaseSpecification: true,
+            today,
+            stdout,
+            stderr,
+          });
+
+          expect(await fileExists(releaseSpecificationPath)).toBe(true);
+        });
+      });
+    });
+
+    describe('when firstRemovingExistingReleaseSpecification is true, the release spec file already exists, an editor is available, and editing will succeed', () => {
```

Comment:
Similarly, is there a way for us to ensure it's following the new release spec here rather than the old one?

Reply:
Yup, makes sense too. Added in [fbc5bf4](https://github.com/MetaMask/create-release-branch/pull/21/commits/fbc5bf415790580d82b360e9971d38a78b63c448).